### PR TITLE
Feat: support isAndroid and isIOS

### PR DESCRIPTION
**Pull request by balloonzzq**
Support isAndroid and isIOS

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -18,16 +18,22 @@ import { isWeex, isWeb, isMiniApp, isReactNative, isNode } from 'universal-env';
 
 ## APIS
 ### `isWeex: boolean`
-校验weex环境
+校验 weex 环境
```
Comment (yacheng) [inline README-zh_CN.md]: Weex 首字母大写

File: README-zh_CN.md [hunk 2]
```diff
@@ -18,16 +18,22 @@ import { isWeex, isWeb, isMiniApp, isReactNative, isNode } from 'universal-env';
...
 
 ### `isWeb: boolean`
-校验web环境
+校验 web 环境
```
Comment (yacheng) [inline README-zh_CN.md]: Web

File: README-zh_CN.md [hunk 3]
```diff
@@ -18,16 +18,22 @@ import { isWeex, isWeb, isMiniApp, isReactNative, isNode } from 'universal-env';
...
 ### `isReactNative: boolean`
-校验React Native环境
+校验 React Native 环境
 
 ### `isNode: boolean`
-校验Node.js环境
+校验 Node.js 环境
+
+### `isAndroid: boolean`
+校验 android 环境
```
Comment (yacheng) [inline README-zh_CN.md]: Android

File: src/index.ts [hunk 4]
```diff
@@ -10,3 +10,28 @@ export const isWeb: boolean = isWebPure && !isMiniApp;
...
+  miniAppSystemInfo = my.getSystemInfoSync();
+}
+
+export const isAndroid = (() => {
+  if (isMiniApp) {
+    return ['Android', 'android'].includes(miniAppSystemInfo.platform);
+  } else if (isWeex) {
+    return navigator.platform.toLowerCase() === 'android';
+  } else {
+    return Boolean(navigator.userAgent.match(/android/i));
```
Comment (yacheng) [inline src/index.ts]: 看下在 node 端调用是否会报错

**Comment by liaohainan** [comment]
return ['iPhone OS', 'iOS'].includes(miniAppSystemInfo.platform);这里的iOS应该是小写ios

**Comment by SoloJiang** [comment]
+1

**Comment by SoloJiang** [comment]
> return ['iPhone OS', 'iOS'].includes(miniAppSystemInfo.platform);这里的iOS应该是小写ios

image

**Pull request merged**